### DISCUSSION TITLE
issue is related to downgrade the airflow version 2.5.1 to 2.3.0

### DISCUSSION THREAD

**Discussion by mehtab123** [airflow-helm/charts#706] [Questions & Answers]
We are trying to downgrade the airflow 2.5.1 to 2.3.0, previous we had the 2.3.0 it upgraded easily using terraform .  

This time when we are deploying,  it is sticking in db migration issue.  Following error is coming on terraform apply -target=helm_release.airflow

```
 Error: failed post-install: timed out waiting for the condition
│
│   with helm_release.airflow,
│   on airflow.tf line 9, in resource "helm_release" "airflow":
│    9: resource "helm_release" "airflow" {
```

Can any one help me regarding this issue

Thanks in advance

**Comment by thesuperzapper** [comment]
@mehtab123 In general, it's a bad idea to downgrade airflow, but if you want to do it, you will need to manually downgrade your airflow db.

Follow the airflow docs for more information: https://airflow.apache.org/docs/apache-airflow/2.5.2/howto/usage-cli.html#downgrading-airflow
